pkg/grpc/api.go

```diff
@@ -107,6 +115,22 @@ func (a *api) CallLocal(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 		return nil, status.Errorf(codes.InvalidArgument, "parsing InternalInvokeRequest error: %s", err.Error())
 	}
 
+	operation := req.Message().Method
+	var httpVerb common.HTTPExtension_Verb
+
+	if req.Metadata() != nil && len(req.Metadata()) > 0 {
```

Comment:
Does this mean this will only work for HTTP enabled Dapr apps?

Reply:
fixed